docs/resources/default_vpc_deletion.md

```diff
@@ -0,0 +1,51 @@
+---
+page_title: "awsutils_default_vpc_deletion Resource - terraform-provider-awsutils"
+subcategory: "Default VPC"
+description: |-
+  
+---
+
+# awsutils_default_vpc_deletion (Resource)
+
+Deletes the default VPC along with the child resources of the VPC including Subnets, Route Tables, NACLs and Internet 
+Gateways in the configured region.
+
+Best-practices call for not using the default VPC, but rather, creating a new set of VPCs as necessary. AWS Security Hub 
+will flag the default VPCs as non-compliant if they aren't configured with best-practices. Rather than jumping through 
+hoops, it's easier to delete to default VPCs. This task cannot be accomplished with the official AWS Terraform Provider, 
+so this resource is necessary. 
+
+Please note that this resource is unusual as it will DELETE infrastrucutre when `teraform apply` is run rather than 
```

Comment:
```suggestion
Please note that this resource is unusual as it will DELETE infrastrucutre when `teraform apply` is run rather than 
```
```suggestion
Please note that this resource is unusual as it will DELETE infrastrucutre when `teraform apply` is run rather than 
```
```suggestion
Please note that applying this resource is destructive and nonreversible. This resource is unusual as it will **DELETE** infrastructure when `terraform apply` is run rather than 
```